custom_components/localtuya/sensor.py

```diff
@@ -47,6 +47,11 @@ def __init__(
     def state(self):
         """Return sensor state."""
         return self._state
+    
```

Comment:
Do we also need to add a last_reset property?
```suggestion
    def last_reset(self):
        return "1970-01-01T00:00:00+00:00"
        
```


As per https://community.home-assistant.io/t/converting-watt-unit-to-kwh-for-individual-devices-energy-dashboard/359730/2

Reply:
Edit: No, not unless it's total.

```
 Setting last_reset for entities with state_class other than 'total' is not supported. Please update your configuration if state_class is manually configured, otherwise report it to the custom component author.
 ```